src/components/Splash/Splash.scss

```diff
@@ -11,7 +11,7 @@
     }
 
     p {
-      margin: 0 auto;
+      margin: 1em auto !important;
```

Comment:
Is this what resolves #1559?

Reply:
Just a part, I just checked, it is still bug at build as a not error in dev mode.
![captursssssssssssssse](https://user-images.githubusercontent.com/12815768/29884336-5e4bf958-8ddd-11e7-8436-6511716cfd63.PNG)
The left side is wrapped by 1 p tag and not the right one

Reply:
About this `margin`, this doesn't resolve this issue, just make a `p` to center correctly. But, there is a difference between `css` priority between the dev and build versions.

Reply:
Ah I see now, yeah that is weird. I think once we start refactoring our markdown parsing process and move to remark a lot of these things will be resolved.

Reply:
I'll see if I can find a temporary fix.